src/mame/drivers/x68k.cpp

```diff
@@ -139,6 +139,8 @@
 #include "x68000.lh"
 
 
+static uint32_t adpcm_clock[2] = { 8000000, 4000000 };
+static uint32_t adpcm_div[4] = { 1024, 768, 512, /* Reserved */512 };
```

Comment:
Static mutable variables are going to break things - can this be made constant, or should it be moved to the state class?